src/main/java/io/jenkins/plugins/oidc_provider/config/ClaimTemplate.java

```diff
@@ -77,6 +79,14 @@ public FormValidation doCheckName(@QueryParameter String value) {
             }
         }
 
+        public FormValidation doCheckRequiredEnvVars(@QueryParameter String value) {
+            if (!value.equals(value.toUpperCase())) {
```

Comment:
can they be null? I think so (and I think they should ) but unclear reading the help inline (thanks for adding it by the way)

Reply:
https://github.com/jenkinsci/stapler/blob/master/core/src/main/java/org/kohsuke/stapler/QueryParameter.java#L55-L60

It wont be null as fixEmpty is false by default

Reply:
Sorry if I wasn't clear enough with my question. I meant if null (or empty) is a valid value or we are missing a validation here. I think it can/should be "nullable", but I'd like to confirm

Reply:
I get what you mean now, I think @jglick suggestion around `Util.fixEmpty` for the setter solves this from the actually setting POV.